src/main/cli/cli.c

```diff
@@ -2410,13 +2410,18 @@ static void cliFlashInfo(const char *cmdName, char *cmdline)
 
     const flashGeometry_t *layout = flashGetGeometry();
 
-    cliPrintLinef("Flash sectors=%u, sectorSize=%u, pagesPerSector=%u, pageSize=%u, totalSize=%u JEDEC ID=0x%08x",
-            layout->sectors, layout->sectorSize, layout->pagesPerSector, layout->pageSize, layout->totalSize, layout->jedecId);
+    cliPrintLinef("%s%u, %s%u, %s%u, %s%u, %s%u, %s=0x%08x", 
```

Comment:
😱 this line is not okay

Reply:
:-), yes but those kind of information are really normally translated in Danish, Franch, maybe German.
But you also an improtant point, that it schould be esay to develop. And it will be, translation can be done later/or maybe never.

Reply:
```
    cliPrintLinef("%s%u, %s%u, %s%u, %s%u, %s%u, %s=0x%08x", 
                  STR_CLI_FLASH_SEC,        layout->sectors, 
                  STR_CLI_FLASH_SEC_SIZE,   layout->sectorSize, 
                  STR_CLI_FLASH_SEC_PAGE,   layout->pagesPerSector, 
                  STR_CLI_FLASH_PAGE_SIZE,  layout->pageSize, 
                  STR_CLI_FLASH_TOTAL_SIZE, layout->totalSize, 
                  STR_CLI_FLASH_JEDEC,      layout->jedecId);
```
All are really subject to translation, in Danish they are translated

Reply:
You can also use (STR_CLI_FLASH_SEC" %u ", " STR_CLI_FLASH_SEC_SIZE "%u" ... 

At least it will reduce flash size